v2/cli/term/page_router.go

```diff
@@ -63,13 +64,15 @@ func (r *pageRouter) loadEventPage(eventID string) {
 	r.loadPage(eventPageName, func() {
 		r.eventPage.refresh(eventID)
 	})
+	go r.eventPage.streamEventLog(eventID)
 }
 
 // loadJobPage refreshes the job page and brings it into focus.
 func (r *pageRouter) loadJobPage(eventID, jobID string) {
 	r.loadPage(jobPageName, func() {
 		r.jobPage.refresh(eventID, jobID)
 	})
+	go r.jobPage.streamLog(eventID, jobID)
```

Comment:
We definitely shouldn't start streaming these logs until the user actually elects to view them.